src/qibolab/instruments/zhinst.py

```diff
@@ -948,6 +927,9 @@ def measure_relax(self, exp, qubits, relaxation_time, acquisition_type):
                 for pulse, q, iq_angle in zip(pulses, qubits, iq_angles):
                     pulse.zhpulse.uid += str(i)
 
+                    # TODO: if the measure sequence starts after the last pulse, add a delay
+                    # if the delay is too large (i.e. 5us),
```

Comment:
Could you check the maximun before it crashes ?

Reply:
Sure.